### DISCUSSION TITLE
How do i add Open Graph ?

### DISCUSSION THREAD

**Discussion by 0xrotense** [YOURLS/YOURLS#3789] [General]
Hello everyone, 

Please, How do i Show Open Graph data from the webpage my short links link to.

This plugin not working anymore, https://github.com/LK608/OG-Scraper

The above plugins shows ERROR 500 after activation.

Thanks & Regards.

**Comment by dgw** [comment]
Examining PHP error logs would have told you that the plugin is instantiating PHPScraper incorrectly. I've opened LK608/OG-Scraper#5 to fix it.

If you didn't even get that far, ensure you followed the instructions to install PHPScraper and point the `require` line in `plugin.php` to the correct location of `autoload.php` after doing so.
